siliconcompiler/tools/klayout/def2stream.py

```diff
@@ -134,6 +134,7 @@ def read_fills(top):
   tech = pya.Technology()
   tech.load(tech_file)
   layoutOptions = tech.load_layout_options
+  layoutOptions.lefdef_config.macro_resolution_mode = 1
```

Comment:
I'm actually not sure, but it's a (semi-)recent change to the OpenROAD-flow-scripts def2stream.py which we used as a source for this file.

IIRC, we opted to copy it and add the original license to the top of the file to avoid requiring a full clone of the ORFS submodule for this single script, and because we made some modifications to pull values from the schema.

Reply:
Sounds good, makes sense to keep up with what they're doing.